src/extensions/scratch3_boost/index.js

```diff
@@ -1926,10 +1934,10 @@ class Scratch3BoostBlocks {
         case BoostColorLabel.ANY:
             if (Object.keys(BoostColor).find(key => BoostColor[key])
                 .toLowerCase() !== this.getColor()) {
-                if (this.getColor() === this._peripheral.oldColor) {
```

Comment:
Was this a bug before and should have been `this._peripheral._sensors.oldColor`? (Just trying to understand the change completely)